docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -18,8 +18,11 @@ customization for lifecycle.
 
 [NOTE]
 ====
-Some older indices might be managed by {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] if a cluster is upgraded to 8.17.
+The indices that were created in 8.15.x and 8.16.x might be managed by {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)].
+More details are available in <<apm-release-notes-8.15>>.
+In 8.17 by default the new indices will be managed by ILM again.
```

Comment:
I removed "again" because we shouldn't assume that someone using 8.17 also used Elastic APM before 8.15 (so this might be the first time their indices are managed by ILM).

```suggestion
Indices created in 8.15.x and 8.16.x might be managed by {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)].
More details are available in <<apm-release-notes-8.15>>.
By default, new indices created in 8.17 are managed by ILM.
```